inference-engine/thirdparty/clDNN/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -6813,6 +6829,142 @@ INSTANTIATE_TEST_SUITE_P(convolution_depthwise_gpu_b_fs_yx_fsv16,
                         ),
                         convolution_depthwise_gpu_fsv16::PrintToStringParamName);
 
+struct convolution_fsv16_to_bfyx_gpu : public ::testing::TestWithParam<TestParamType_convolution_bfyx_padding_gpu> {
+    static std::string PrintToStringParamName(
+        testing::TestParamInfo<TestParamType_convolution_bfyx_padding_gpu> param_info) {
+        // construct a readable name
+        std::string res = "in" + std::to_string(testing::get<0>(param_info.param)) + "b" +
+            std::to_string(testing::get<1>(param_info.param)) + "f" +
+            std::to_string(testing::get<2>(param_info.param)) + "y" +
```

Comment:
the order is not matching well.. b is param[0] and f is parm[1]. ..

Reply:
Fix to use convolution_general_gpu